docs/src/development.md

```diff
@@ -6,11 +6,13 @@ This document is aimed at *Quality-time* developers and maintainers and describe
 
 ### Running *Quality-time* locally
 
-When developing *Quality-time*, we recommend running the bespoke components from a shell and the standard components in Docker. The server and frontend components have auto-reload, meaning that when you edit the code, they will restart and run the new code automatically. The collector and notifier components don't have auto-reload, and need to stopped and started by hand to activate new code.
+When developing *Quality-time*, there are two ways to run *Quality-time* locally: in Docker completely (scenario 1 below) or partly in Docker and partly from shells (scenario 2 below).
+
+We recommend scenario 2: running the bespoke components from a shell and the standard components in Docker. The server and frontend components have auto-reload, meaning that when you edit the code, they will restart and run the new code automatically. The collector and notifier components don't have auto-reload, and need to stopped and started by hand to activate new code.
```

Comment:
I would be useful to let the user know which components are meant with the term 'bespoke'. I would suggest the first time you use the word 'bespoke' sum the components up in parentheses.